developer/src/kmc-package/src/compiler/package-version-validation.ts

```diff
@@ -73,7 +73,7 @@ export class PackageVersionValidation {
 
     if(result) {
       if(followKeyboardVersion) {
-        kmp.info.version.description = kmp.keyboards[0].version;
+        kmp.info.version = {description: kmp.keyboards[0].version};
```

Comment:
Sounds reasonable.  Part of me wonders if there ever would be an existing definition with other members we may wish to save, but realistically speaking... I severely doubt it, in which case this is fine.

Reply:
Yeah, this is actually better than what we had previously -- the `followKeyboardVersion` flag says we should ignore metadata on version found in the package, so this matches the intention of that flag more closely.

In any case, the only other metadata available in `kmp.info.version` is `.url` which is a nonsense anyway for `.version` (yay generic structures for metadata, bad design decision by me way back in the past. The `.url` property is used for some of the other `.info` properties such as `author` -- e.g. `mailto:me@example.com`).